src/app/frontend/replicasetdetail/replicasetdetail.scss

```diff
@@ -102,6 +97,26 @@ $table-cell-height-half: $table-cell-height / 2;
   width: 15 * $baseline-grid;
 }
 
+.kd-replicasetdetail-tab-header {
+  md-tabs-canvas,
+  md-pagination-wrapper {
+    height: $toolbar-height-size-base;
```

Comment:
In my opinion it looks better when we don't change height here (use standard Angular Material), but color should definitely be changed here (is it pink?) :)